modules/data.remote/DESCRIPTION

```diff
@@ -12,7 +12,8 @@ Imports:
     reticulate,
     PEcAn.logger,
     PEcAn.remote,
-    stringr (>= 1.1.0)
+    stringr (>= 1.1.0),
+    binaryLogic
```

Comment:
Is binaryLogic needed for the QA/QC handling?

Reply:
I think so. This was already in the DESCRIPTION file before I messed around with the function. Most of the others have this also.

Reply:
Not a showstopper, but consider moving binaryLogic up the list to maintain alphabetical order for easier reading. (None of the scripts that parse imports care, this is strictly for the convenience of human reader.)